### ISSUE TITLE
Loss and Gradient Normalization is zero While performing full finetuning of Qwen2.5-32B

### ISSUE THREAD

**Issue by mohit-217** [axolotl-ai-cloud/axolotl#2357]
### Please check that this issue hasn't been reported before.

- [x] I searched previous [Bug Reports](https://github.com/axolotl-ai-cloud/axolotl/labels/bug) didn't find any similar reports.

### Expected Behavior

There should be some loss and gradient Normalization while performing training.

### Current behaviour

{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 4.0000000000000003e-07, 'epoch': 0.05}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 6.000000000000001e-07, 'epoch': 0.08}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 8.000000000000001e-07, 'epoch': 0.1}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 1.0000000000000002e-06, 'epoch': 0.13}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 1.2000000000000002e-06, 'epoch': 0.15}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 1.4000000000000001e-06, 'epoch': 0.18}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 1.6000000000000001e-06, 'epoch': 0.2}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 1.8000000000000001e-06, 'epoch': 0.23}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 2.0000000000000003e-06, 'epoch': 0.25}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 2.2e-06, 'epoch': 0.28}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 2.4000000000000003e-06, 'epoch': 0.3}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 2.6e-06, 'epoch': 0.33}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 2.8000000000000003e-06, 'epoch': 0.35}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 3e-06, 'epoch': 0.38}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 3.2000000000000003e-06, 'epoch': 0.4}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 3.4000000000000005e-06, 'epoch': 0.43}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 3.6000000000000003e-06, 'epoch': 0.45}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 3.8000000000000005e-06, 'epoch': 0.48}

### Steps to reproduce

Just launch training with below config file
accelerate launch -m axolotl.cli.train new_qwen.yaml

### Config yaml

```yaml
base_model: Qwen/Qwen2.5-32B
# Automatically upload checkpoint and final model to HF
# hub_model_id: username/custom_model_name
# model_type: AutoModelForCausalLM
# tokenizer_type: AutoTokenizer
load_in_8bit: false
load_in_4bit: false
strict: false
chat_template: qwen_25
datasets:
  - path: cognitivecomputations/dolphin-r1
    type: chat_template.default
    #chat_template: chatml
    field_messages: messages
    message_property_mappings:
      role: role
      content: content
    roles:
      system:
        - system
      user:
        - user
      assistant:
        - assistant
    data_files:
       - dolphin-r1-reasoning-deepseek.jsonl
    trust_remote_code: True
dataset_prepared_path: last_run_prepared
val_set_size: 0.05
output_dir: ./outputs/out

sequence_len: 8192
sample_packing: false
pad_to_sequence_len: true

wandb_project:
wandb_entity:
wandb_watch:
wandb_name:
wandb_log_model:

gradient_accumulation_steps: 2
micro_batch_size: 1
num_epochs: 1
optimizer: adamw_torch
lr_scheduler: cosine
learning_rate: 2e-5

train_on_inputs: false
group_by_length: false
bf16: auto
fp16:
tf32: false

gradient_checkpointing: true
# gradient_checkpointing_kwargs:
#   use_reentrant: false
early_stopping_patience:
resume_from_checkpoint:
logging_steps: 1
xformers_attention:
flash_attention: true

warmup_steps: 100
evals_per_epoch: 2
eval_table_size:
saves_per_epoch: 1
debug:
deepspeed: deepspeed_configs/zero3_bf16_cpuoffload_params.json
weight_decay: 0.0
fsdp:
fsdp_config:
#special_tokens:
  #pad_token: <|end_of_text|>
```

### Which Operating Systems are you using?

- [x] Linux
- [ ] macOS
- [ ] Windows

### Python Version

3.10.12

### axolotl branch-commit

main

### Acknowledgements

- [x] My issue title is concise, descriptive, and in title casing.
- [x] I have searched the existing issues to make sure this bug has not been reported yet.
- [x] I am using the latest version of axolotl.
- [x] I have provided enough information for the maintainers to reproduce and diagnose the issue.

**Comment by NanoCode012** [comment]
Hey, have you tried with our default dataset config? Do the results also appear like that? What system is this on (gpus, torch version)?

**Comment by mohit-217** [comment]
I have resolved this issue there was some issues related to custom dataset integration.

**Comment by NanoCode012** [comment]
Thanks for letting us know!

**Comment by shing100** [comment]
@mohit-217 Hello, I'm experiencing the same problem, can I ask you to share how you solved it?
